packages/legend-application-studio/src/stores/editor/EditorStore.ts

```diff
@@ -926,6 +929,133 @@ export class EditorStore implements CommandRegistrar {
     }
   }
 
+  getSupportedElementTypesWithCategory(): Map<string, string[]> {
```

Comment:
cant this be a calculated attribute and created when editorstore is created? seems like we are computing this upon each rerender